tasks/__init__.py

```diff
@@ -86,11 +86,13 @@ def git_logs(ctx, branch=None):
 
 
 @task
-def apiserver(ctx, port=8000, wait=True, host='127.0.0.1'):
+def apiserver(ctx, port=8000, wait=True, autoreload=True, host='127.0.0.1'):
```

Comment:
I'm not certain that it's possible to specify a `False` value for boolean flags for `invoke`. I remember running into this before, and a quick grep through [their docs](http://docs.pyinvoke.org/en/0.13.0/index.html) doesn't help much.
```
~/D/osf.io ❯ inv apiserver --autoreload=0                      ⏎ remotes/cos/pr/6104 ✭ ◼
No idea what '0' is!
~/D/osf.io ❯ inv apiserver --autoreload=False                  ⏎ remotes/cos/pr/6104 ✭ ◼
No idea what 'False' is!
~/D/osf.io ❯ inv apiserver --autoreload=null                   ⏎ remotes/cos/pr/6104 ✭ ◼
No idea what 'null' is!
~/D/osf.io ❯ inv apiserver --autoreload=false                  ⏎ remotes/cos/pr/6104 ✭ ◼
No idea what 'false' is!
```
Consider a default of `noreload=False` instead